### ISSUE TITLE
Add the 'exportprivatekey' function to the file menu.

### ISSUE THREAD

**Issue by bocyaj** [NAVCoin/navcoin-core#120]
Rename "list old addresses" button to "used addresses"

Add export private key option to the right click menu of the address list.

Check if the wallet is unlocked and display the password form if locked (look at import private key option in menu to see how this can be done).

Display the public and private key to the user if successful in a text popup box.

- - -
[Elegantt data. What's this?](http://bit.ly/elegantt-for-trello-whats-this)
[](Elegantt!UNITO-UNDERSCORE!data:dont!UNITO-UNDERSCORE!delete{"ignored":false,"autoPlanned":true,"ownerId":false,"dependencies":[{"hidden":false,"from":"UqSz4Iih"}],"psd":"2018-01-25T05:34:00+13:00","ped":"2018-01-25T22:42:00+13:00"})

**Comment by navtrello** [comment]
➤ Craig MacGregor commented:

Original Ticket: 

I am seeing quite a few issues where users are having difficulty dumping their private keys using the command line.  It isn't completely apparent that you need to unlock your wallet first using 'walletpassphrase' before importing or exporting your keys.  Using command line also leads to the wallet  being unlocked for X amount of seconds due to how walletpassphrase works.

The 'import private key function' found under the file menu on main UI is perfect, as it prompts you to enter the passphrase and does not affect the current state of the wallet, much the same when transfering coins.

I would like to request that the 'export private key' function be added to this menu in the same way.

Thanks.

**Comment by navtrello** [comment]
➤ Craig MacGregor commented:

@matt36511783 is this blocked until we get the Core dev container setup? Or something else?

**Comment by navtrello** [comment]
➤ Matt commented:

@craigmacgregor2 Yeah waiting on the container. I mean I can try get it compiling myself but then we have two people (myself and Paul) trying to get one thing compiling and it's probably not an efficient use of time

**Comment by navtrello** [comment]
➤ Matt commented:

Though I'll probably still dig through the code and plan how to do what I want to do

**Comment by navtrello** [comment]
➤ Craig MacGregor commented:

Sure, when something is blocked by another task, add it as a dependency like ive done here. Or if it's blocked by something that's not in Trello, comment on why it's moved to blocked :)

**Comment by aguycalled** [comment]
2a28fb1

**Comment by navtrello** [comment]
➤ Craig MacGregor commented:

moving this back out from Done -> Q/A 

@matt36511783 your task is now to review this commit and make sure it meets the acceptance criteria once we get the docker dev environment setup.

**Comment by navtrello** [comment]
➤ Matt commented:

Works as spec'd
